docs/tutorials/cheatsheet.rst

```diff
@@ -96,7 +96,7 @@ This is related to `Issue 42 <https://github.com/thu-ml/tianshou/issues/42>`_.
 
 If you want to get log stat from data stream / pre-process batch-image / modify the reward with given env info, use ``preproces_fn`` in :class:`~tianshou.data.Collector`. This is a hook which will be called before the data adding into the buffer.
 
-This function receives typically 7 keys, as listed in :class:`~tianshou.data.Batch`, and returns the modified part within a dict or a Batch. For example, you can write your hook as:
+This function receives typically up to 7 keys, as listed in :class:`~tianshou.data.Batch`, and returns the modified part within a :class:`~tianshou.data.Batch`. If it receives only ``obs``, it is applied in the env reset; otherwise, it is applied in the normal env step. For example, you can write your hook as:
```

Comment:
This part is not clear. I would rather all the keys instead of just saying `typically up to 7 keys`. And I would remove `typically`. Than I would replace ` If it receives only ``obs``, it is applied in the env reset; otherwise, it is applied in the normal env step.` by `only ``obs`` is defined at env reset, while every key are specified for normal steps.`